pages/api/program/index.ts

```diff
@@ -30,12 +30,22 @@ export default async function handle(req: NextApiRequest, res: NextApiResponse):
             if (validationError.length !== 0) {
                 ResponseUtil.returnBadRequest(res, validationError.join(", "));
             } else {
-                const newProgram = await createProgram(req.body as ProgramInput);
-                if (!newProgram) {
-                    ResponseUtil.returnBadRequest(res, `Program could not be created`);
-                    break;
+                try {
+                    const newProgram = await createProgram(
+                        req.body.programData as ProgramInput,
+                        req.body.translationData as ProgramTranslationData[],
+                    );
+
+                    if (!newProgram) {
+                        ResponseUtil.returnBadRequest(res, `Program could not be created`);
+                        break;
+                    }
+
+                    ResponseUtil.returnOK(res, newProgram);
+                } catch (e) {
+                    console.error(e);
+                    ResponseUtil.returnBadRequest(res, `Internal Error: ` + e);
                 }
```

Comment:
I don't think we need a try/catch here unlike for the stripe price? If we get an internal error, we get an internal error and should not return it as a bad request response.

Reply:
I'm going to remove the try/catch and add a TODO to verify the translation input.